### PR TITLE
Added info page for how to share code

### LINKED ISSUE #3: Compile resources for how to share code on Discord
One page I really want to have on this site is a how-to for various ways to share code on Discord.

I'd like to cover the following, so let's start by gathering screenshots, etc., for these:

- Single lines of code
- Codeblocks (and syntax highlighting)
- Sharing Gists directly from Codecademy
- Sharing code screenshots from https://carbon.now.sh/
- Uploading an entire code file

### PR DESCRIPTION
**Pull request by lyallstewart**
## What issue is this solving?

Closes #3 

### Description
A page with some of ways people can share code on the server.

### Please make sure you've attempted to meet the following coding standards

- [X] Code has been tested and does not produce errors
- [X] Code is readable and formatted
- [X] There isn't any unnecessary commented-out code

### CODE REVIEW

File: sharing_code.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet"> 
+    <link href="src/css/style.css" rel="stylesheet" type="text/css">
```
Comment (Victoria-DR) [inline sharing_code.html:11]: I think it'd be better to import all of the fonts we'll be using in `style.css`.

File: sharing_code.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                <p>If you need to export code directly from the Codecademy workspace, you can click on the tools header on the navbar and export your code to a gist:</p>
+                <img src="./src/img/cc_sharing.png">
+            </li>
+            <li>
+                <h3>Sharing off-platform code</h3>
+                <p>To share off platform code you can use a code-sharing site. Some of these are:</p>
+                <a href="https://gist.github.com/">Github Gist</a>
+                <a href="https://codepen.io/">Codepen</a>
+                <a href="https://jsfiddle.net/">JSFiddle</a>
+                <p>Or you could share a link to a github repository with your code.</p>
```
Comment (Victoria-DR) [inline sharing_code.html:44]: `github` -> `GitHub`

File: sharing_code.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        <ul>
+            <li>
+                <h3>Exporting Code from Codecademy</h3>
+                <p>If you need to export code directly from the Codecademy workspace, you can click on the tools header on the navbar and export your code to a gist:</p>
+                <img src="./src/img/cc_sharing.png">
+            </li>
+            <li>
+                <h3>Sharing off-platform code</h3>
+                <p>To share off platform code you can use a code-sharing site. Some of these are:</p>
+                <a href="https://gist.github.com/">Github Gist</a>
```
Comment (Victoria-DR) [inline sharing_code.html:41]: `Github` -> `GitHub`

File: sharing_code.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        <img src="./src/img/snippet.png">
+        <p>This has a number of benefits such as improved readability and syntax highlighting.</p>
+    </div>
+    <div>
+        <h2>Larger Code Files</h2>
+        <p>If you need to send a larger code snippet or a whole file, please avoid uploading it directly as a file and instead use one of these options:</p>
+        <ul>
+            <li>
+                <h3>Exporting Code from Codecademy</h3>
+                <p>If you need to export code directly from the Codecademy workspace, you can click on the tools header on the navbar and export your code to a gist:</p>
```
Comment (Victoria-DR) [inline sharing_code.html:35]: `tools` -> `Tools`

File: sharing_code.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                <p>To share off platform code you can use a code-sharing site. Some of these are:</p>
+                <a href="https://gist.github.com/">Github Gist</a>
+                <a href="https://codepen.io/">Codepen</a>
+                <a href="https://jsfiddle.net/">JSFiddle</a>
+                <p>Or you could share a link to a github repository with your code.</p>
+            </li>
+        </ul>
+    </div>
+
+</body>
```
Comment (Victoria-DR) [inline sharing_code.html:49]: Missing a closing `</html>` tag.

**Review by Victoria-DR — COMMENTED** [review]
Again, this is great! Added comments for minor edits.

**Comment by lyallstewart** [comment]
Thanks @Victoria-DR 
I've just updated those 😀

**Review by Victoria-DR — APPROVED** [review]
LGTM!

**Comment by lyallstewart** [comment]
Awesome, thanks! Would you mind merging it? I don't have the perms to do it myself :)

**Comment by Victoria-DR** [comment]
> Awesome, thanks! Would you mind merging it? I don't have the perms to do it myself :)

I don't have the perms to do so either, looks like we're gonna have to wait on @Vic-ST or @aedwardg.

**Review by Vic-ST — APPROVED** [review]
Whoop! Looks good!

**Pull request merged**